scalableminds/webknossos
# Add (minimal) UI for auto brush and allow it for white-listed datasets

This PR adds
- a configurable white-list (via feature flags)
- a minimal UI (a checkbox) for the auto-brush when a tracing is opened for a whitelisted dataset
- that the auto-brush mode can be disabled by hitting the escape key

The checkbox itself rather serves the purpose of telling the user about the feature (since it would be not easy to find, otherwise). Maybe the checkbox should be activated by default for whitelisted datasets? And an information icon next to the checkbox could give more insights?

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing for the dsA dataset (whitelisted by default in the application.conf)
- the tracing settings should show a checkbox for the autobrush
- when enabling that checkbox a notification should open which tells the user about the ctrl+click usage
- after using ctrl+click, the auto-brush mode should be cancellable by hitting the escape key

### Issues:
- fixes #4042 
- fixes #4004 

------
